packages/app/src/files-manager/components/files-summary/files-summary.component.tsx

```diff
@@ -96,13 +69,13 @@ const FilesSummary: FunctionComponent<Props> = ({
         message={messages.summaryTitle}
       />
       <FilesSummaryWrapper data-testid={FilesSummaryTestIds.Wrapper}>
-        {filteredData.map((box, index: number) => (
+        {data.map((box, index: number) => (
```

Comment:
🟢
I noticed the problem with variable naming... (data, box, memory... too many synonyms)

`data` variable is comes from `DiskSpaceCategory` interface. `diskSpaceCategory` is more meaningful name or `diskSpaceCategories` to mark that is a list.

Generics like `data` aren't so bad. However, it isn't a good example because here is too many relations to these properties (iteration and mapper logic)

```suggestion
        {diskSpaceCategories.map((diskSpaceCategory, index) => (
```

Reply:
Changed in [ce40a64](https://github.com/mudita/mudita-center/pull/598/commits/ce40a6429870732cb157448924ac3de654ac19c3)